lib/schema/provider.ts

```diff
@@ -2,9 +2,13 @@ import * as Joi from 'joi';
 import { Regexes } from '../regexes';
 
 export const ProviderSchema = Joi.object().keys({
+  id: Joi.any(),
+  created_at: Joi.any(),
+  updated_at: Joi.any(),
```

Comment:
Is there a reason we can't make these anything more strict then `any`?

Reply:
@peteplays I thought providers is almost identical to clinics, I'm not sure we should be allowing id, created_at and updated_at ?

Reply:
@jamescgarrett thats a good point, we should probably remove them completely.